### ISSUE TITLE
Vizy seems to ignore current site for entry selection

### ISSUE THREAD

**Issue by maxstrebel** [verbb/vizy#176]
### Describe the bug

If you have a multisite, Vizy will display entries from the default site.

Example: I have a categories section which saves entries to the site group. So categories from site group A will differ from site group B.

When I'm on a site in Site group B, an entry block in the vizy field will only show categories from Group A, although I'm in group B. This might also be the reason for https://github.com/verbb/vizy/issues/132.

### Steps to reproduce

1. Create 2 Sites - Site A, Site B
2. Create a section "Labels" and select "save entries to the site they were created in"
3. Create an Entry in site A, called Label A
4. Create an Entry in site B, called Label B
5. Create a Field "Label Select", that allows to select Entries from "Labels"
6. Create a Vizy Field and Add a Block that uses the "Label Select" field.
7. Add it to a section
8. Create a new Entry in Site B and in the Vizy Field open the select overlay
9. You will see the "Label A", but not Label B.

This shows even more, when you have 2 Labels in Site B - as you will not be able to select the right entry.

### Craft CMS version

Craft Solo 4.2.4

### Plugin version

2.0.6

### Multi-site?

Yes

**Comment by engram-design** [comment]
I'm afraid I'm not seeing this behaviour sorry! When I open the entry with the Vizy field in the second site, and add a Vizy block that contains an entries field, the current site is always selected in the site-selection dropdown.

Also ensure that your entries field has the `Show the site menu` setting enabled (Under "Advanced"), but also note the warning `Relations don’t store the selected site, so this should only be enabled if some entries aren’t propagated to all sites.`.

**Comment by maxstrebel** [comment]
Hey Josh, thanks for the quick reply. When I activate `Show the site menu` it actually works. It will select the site I'm currently in and show the entries.

But when it's deactivated, it will show entries from another site. I added the same field - one time directly to the entry, one time in vizy, one time in NEO. And only Vizy shows the entries from the other site when the site menu is deactivated.

I will use the dropdown as a quick fix, but I think when the site select is deactivated - the current behavior is unexpected.

**Comment by engram-design** [comment]
No, that's a good point, and you're right. Should be fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-4 as 2.0.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by maxstrebel** [comment]
Thanks a lot @engram-design - that fixes it!

**Comment by engram-design** [comment]
Fixed in [2.0.7](https://github.com/verbb/vizy/releases/tag/2.0.7)
